pom.xml

```diff
@@ -529,6 +529,7 @@ under the License.
                         <exclude>release/**</exclude>
                         <!-- antlr grammar files -->
                         <exclude>paimon-common/src/main/antlr4/**</exclude>
+                        <exclude>paimon-core/src/test/resources/compatibility/**</exclude>
```

Comment:
why add this?

Reply:
If not, the rat-plugin always report license warning when building locally as the following:
[WARNING] Files with unapproved licenses:
  paimon-core/src/test/resources/compatibility/datasplit-v1
  paimon-core/src/test/resources/compatibility/datasplit-v2